src/assets/i18n/en.json

```
@@ -1,44 +1,98 @@
 {
+  "ABBREVIATED_NAME_OF_THE_ORGANIZATION": "Abbreviated name of the organization",
   "ABOUT_THE_PORTAL": "ABOUT THE PORTAL",
+  "ACTUAL_ADDRESS": "Actual address",
   "ADD": "Add",
+  "ADD_ACHIEVEMENTS": "Add Achievements",
```

"ADD_ACHIEVEMENTS": "Add Achievements",
```suggestion
  "ADD_ACHIEVEMENT": "Add Achievement",
```